src/pytorch_lightning/utilities/model_summary.py

```diff
@@ -241,7 +268,12 @@ def model_size(self) -> float:
         return self.total_parameters * self._precision_megabytes
 
     def summarize(self) -> Dict[str, LayerSummary]:
-        summary = OrderedDict((name, LayerSummary(module)) for name, module in self.named_modules)
+        all_parameter: List[Tuple[str, Union[nn.Module, nn.Parameter]]]
+        all_parameter = []
+        all_parameter.extend(self.named_modules)
+        all_parameter.extend(self.named_parameters)
```

Comment:
are you sure this isn't traversing recursively?

Reply:
I used 'reverse', so it will not traverse recursively.